eris_test.go

```diff
@@ -35,17 +35,17 @@ func TestErrorWrapping(t *testing.T) {
 		"standard error wrapping with global root cause (eris.NewGlobal)": {
 			cause:  globalErr,
 			input:  []string{"additional context", "even more context"},
-			output: "global error: additional context: even more context",
+			output: "global error:additional context:even more context",
```

Comment:
this doesn't look very good. any way we can still have a space when the stack is not included?

Reply:
hmm, have to pass different formatter when trace is false. I will try